ui/appui-layout-react/src/appui-layout-react/preview/PreviewFeatures.tsx

```diff
@@ -23,6 +23,11 @@ export interface KnownPreviewLayoutFeatures {
    * Discuss or upvote this feature: https://github.com/iTwin/appui/discussions/673
    */
   enableMaximizedFloatingWidget: boolean;
+  /** If true, a tab, or the active tab of a group of wiget will become active when dropping them in a container.
```

Comment:
I typoed in my initial suggestion.
```suggestion
  /** If true, a tab, or the active tab of a group of widget will become active when dropped in a container.
```